src/main/java/hudson/plugins/git/GitTool.java

```diff
@@ -137,7 +138,7 @@ public static void onLoaded() {
     }
 
 
-    @Extension
+    @Extension @Symbol("git")
```

Comment:
Right now, nothing will see it at all, but as I mentioned on the JIRA, this will enable Pipeline or Job DSL to refer to `GitTool` without needing the full package and class name, etc. I'm fine with "cli-git" or "git" or whatever - I just want a `@Symbol`. =)